src/libraries/System.Text.RegularExpressions/tests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -30,10 +30,14 @@
     <Compile Include="RegexMatchTimeoutExceptionTests.cs" />
     <Compile Include="RegexParserTests.cs" />
     <Compile Include="RegexReductionTests.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
```

Comment:
Oh wait, we should only include the .netfx.cs file here. RegexParseError.cs should be in NetCoreAppCurrent I think. Double checking locally